PowerEditor/src/NppNotification.cpp

```diff
@@ -743,6 +743,11 @@ BOOL Notepad_plus::notify(SCNotification *notification)
 					// which are closest to the clicked position.
 					for (int32_t i = static_cast<int32_t>(position_of_click); i >= 0; --i)
 					{
+						if (bufstring.empty()) 

```

Comment:
I would rather do such check earlier, right after I have got the `bufstring` content.
E.g. on the line 739: 
```
if (bufstring.empty())
	break;
```